client/state/sites/plans/selectors.js

```diff
@@ -98,6 +118,32 @@ export function getPlanDiscountedRawPrice(
 }
 
 /**
+ * Returns a plan price before discount
+ *
+ * @param  {Object}  state         global state
+ * @param  {Number}  siteId        the site id
+ * @param  {String}  productSlug   the plan product slug
+ * @param  {Boolean} isMonthly     if true, returns monthly price
+ * @return {Number}                plan raw price
+ */
+export function getPlanRawPrice(
```

Comment:
I wonder.. shouldn't we prefix these selectors with `site`? Like `getSitePlanRawPrice`? I know it's a lengthy name, but:

1. other selectors in this file do it as well (ie `getSitePlan`);
2. it makes sense to distinguish between selectors used for `state/plans` and `state/sites/plans` because those endpoints return different things;
3. there might be naming collisions if we import `getPlanRawPrice` from `state/plans/selectors` and this one.

I know we can still import these `as` something with different name (ie `import { getPlanRawPrice as getSitePlanRawPrice }`) but who will do that, right?

On the other hand, programmers need to learn a difference between site plans and plans before they can understand these selectors so maybe it's not necessary.

What do you think?

Note: the same goes for `isPlanDiscounted`.